src/sentry/search/utils.py

```diff
@@ -171,6 +171,23 @@ def tokenize_query(query):
             results['query'].append(token)
             continue
 
+        # this handles quoted string, and is duplicated below
+        if token[0] == '"':
+            nvalue = token
+            while not nvalue.endswith('"'):
```

Comment:
Can we do `nvalue[-1:] == '"'` plz since it's significantly faster.

```
$ python -m timeit -s 'v = "x" * 1000' 'v[-1:] == "x"'
10000000 loops, best of 3: 0.0747 usec per loop
$ python -m timeit -s 'v = "x" * 1000' 'v.endswith("x")'
1000000 loops, best of 3: 0.228 usec per loop
```